client/state/memberships/settings/actions.js

```diff
@@ -2,11 +2,47 @@
  * Internal dependencies
  */
 
-import { MEMBERSHIPS_SETTINGS } from 'state/action-types';
+import {
+	MEMBERSHIPS_SETTINGS,
+	MEMBERSHIPS_CONNECTED_ACCOUNTS_STRIPE_DISCONNECT_SUCCESS,
+	NOTICE_CREATE,
+} from 'state/action-types';
 
 import 'state/data-layer/wpcom/sites/memberships';
+import wpcom from 'lib/wp';
 
 export const requestSettings = siteId => ( {
 	siteId,
 	type: MEMBERSHIPS_SETTINGS,
 } );
+
+export const requestDisconnectStripeAccount = ( siteId, connectedAccountId, noticeText ) => {
+	return dispatch => {
```

Comment:
/disconnect takes a really long time, because it loops through all my other sites.

Reply:
Good idea!